app/system/controllers/Extensions.php

```diff
@@ -31,7 +31,7 @@ class Extensions extends \Admin\Classes\AdminController
         ],
     ];
 
-    protected $requiredPermissions = 'Admin.Extensions';
```

Comment:
would having both Admin.Extensions and Site.System work here?

Reply:
that way admin with any of the permission would have access

Reply:
Yep that allows users to with Site.Settings to be able to manage the setting for the extension, but would then allow anyone with Site.Settings to amend extensions (if they go directly to the URL), so the `if (!$this->getUser()->hasPermission('Admin.Extensions')) ` or some version of that is needed

Reply:
I think the if check works for what you want, my only concern was with using `Admin.*`